workspace/solution.go

```diff
@@ -67,11 +71,15 @@ func (s *Solution) Write(dir string) error {
 		return err
 	}
 
-	path := filepath.Join(dir, solutionFilename)
+	path, err := createIgnoreSubdir(dir)
+	if err != nil {
+		return err
+	}
 
 	// Hack because ioutil.WriteFile fails on hidden files
 	visibility.ShowFile(path)
```

Comment:
So we are calling ShowFile on a directory then down below we are returning the result of visibility.HideFile on the full path to the solution file. 

At this point I wonder if it is needed since it is now the directory that is a dot file.